packages/browser/src/integrations/globalhandlers.ts

```diff
@@ -166,8 +170,8 @@ function _enhanceEventWithInitialFrame(event: Event, url: any, line: any, column
   // event.exception.values[0].stacktrace.frames
   const ev0sf = (ev0s.frames = ev0s.frames || []);
 
-  const colno = isNaN(parseInt(column, 10)) ? undefined : column;
-  const lineno = isNaN(parseInt(line, 10)) ? undefined : line;
+  const colno = column;
+  const lineno = line;
```

Comment:
m: why are we removing the `isNaN` check here?

Reply:
because it should not actually be needed, I think? `window.onerror = function()` should be supported everywhere in all the browser we support, and is typed consistently, so these should be proper numbers (or undefined) always?